Makefile

```diff
@@ -71,7 +71,7 @@ init:                       ## Install linters
 	cd tools && go generate -x -tags=tools
 
 build:                      ## Compile using plain go build
-	CGO_ENABLED=1 go build -ldflags="$(GO_BUILD_LDFLAGS)"  -o $(PMM_RELEASE_PATH)/mongodb_exporter -tags gssapi
+	go build -ldflags="$(GO_BUILD_LDFLAGS)"  -o $(PMM_RELEASE_PATH)/mongodb_exporter
```

Comment:
this revert PR could include only this as everything else doesn't affect PMM really

Reply:
We want to still keep GSSAPI support in goreleaser builds? @BupycHuk

Reply:
I imagine we can drop it too, and in that case drop the tests as well (which removes the need for the other files).

We still have the original PR, so if we need to prepare some builds, we can base them on that.

Reply:
We can have additional build of mongodb_exporter with gssapi